src/common/components/Tabs/Tabs.d.ts

```diff
@@ -0,0 +1,15 @@
+import {ReactChildren, ReactNode} from 'react';
+
+interface ITab {
+	title: string | ReactChildren | ReactNode;
+	content: any;
+}
+
+export default interface ITabsProps {
+	style?: {
```

Comment:
maybe easier way to set classes for activeTab and another tabs? it's makes component easier, it's set a two string - easy way for comparing props, and remove undefined hell at render with style calculation